react/components/Input/index.js

```diff
@@ -262,7 +263,12 @@ class Input extends Component {
           )}
         </div>
         {errorMessage && (
-          <div className="c-danger t-small mt3 lh-title">{errorMessage}</div>
+          <div className="c-danger flex t-small mt3 lh-title">
+            <span className="mr2 flex items-center">
```

Comment:
This doesn't need to be a `flex`, you can add the `item-scenter` to the parent div.

Reply:
You said to use `flex items-center` on div and remove `flex items-center` from span, right? I tried to use this way but the result wasn't the same.

Reply:
SVG element is `display: inline`,  so line-height and font-size affect it. If I use `display: flex` Svg display will be updated to `block` and won't be affected by line-height and font-size.